src/components/panel/panel.js

```diff
@@ -637,7 +660,9 @@ var FOCUS_TRAP_TEMPLATE = angular.element(
 
 
 /**
+ * @description
```

Comment:
@ErinCoughlan I just thought that all documentation / comment areas should follow the same format. I will remove the `@description` tags.